docs/middleware/response/json.md

```diff
@@ -0,0 +1,29 @@
+---
+layout: documentation
+title: "Json Response Middleware"
+permalink: /middleware/json-response
+hide: true
+prev_name: Instrumentation Middleware
+prev_link: ./instrumentation
+next_name: Logger Middleware
+next_link: ./logger
+top_name: Back to Middleware
+top_link: ./list
+---
+
+The `Json Response` middleware parses response body into a hash of key/value pairs.
```

Comment:
Should this be referred  to like Json::Response here?

Reply:
The class is technically `Faraday::Response::Json`, I could go with that or just `JSON` and move response outside as an adjective to "middleware"

Reply:
I went for the latter (and applied the same change to the request one), but happy to change if you prefer the other